features/old/accounts/invoices.feature

```diff
@@ -36,3 +36,37 @@ Feature: Show invoices from account's page (#16015909)
     And I follow "2011-01-00000001"
     # TODO: Properly check if order is opened
     Then I should see "Add"
+
+  Scenario: Display years of invoices when there are invoices
+    Given a provider is logged in on 1st January 2011
+    Given a buyer "zoidberg" signed up to provider "foo.3scale.localhost"
+    And the provider is charging its buyers in prepaid mode
+    And an invoice of buyer "zoidberg" for January, 2011
+    And I go to the invoices issued by me
+    Then I should see the list of years with invoices have the following years:
+      | 0  |
+      | 2011 |
```

Comment:
For this step I would add another invoice for a different year, e.g. 2016 (you may need to do some time machine trick to move to a different year, because there is a validation on invoice period that does not allow creating invoices too far into the future), and confirm that the list of the years is the range between those, in descending order, e.g. 

```
2016
2015
2014
2013
2012
2011
```

Reply:
yes 
` Billing Period must be between the provider account creation date and 12 months from now `

and so I have added 2 invoices which seem to test what we want